### ISSUE TITLE
Medallia change request - Exclude A11y intercept survey from appearing when login modal is open

### ISSUE THREAD

**Issue by aubreyarcangel** [department-of-veterans-affairs/va.gov-team#90807]
## Issue Description
This issue was brought to us by the Identity team. There is an accessibility bug that occurs when the a11y feedback appears while the login modal is open. The feedback modal pops up over the login modal, but none of the call-to-action buttons are responsive because the focus is set on the login modal. The only way to exit is to refresh the browser.

[Private Zenhub Video](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNjVmQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8d7f2162bf17509c36d2d9bc67de01b081b1cb00/Screen%20Recording%202024-08-08%20at%209.23.16%E2%80%AFAM.mov)

Since the login modal can appear on any URL on VA.gov, the best way to exclude the intercept survey is to check if the URL contains the query string `?next=loginModal`

---
## Tasks
- [x] Put in an exemption request to VSignals
- [ ] QA intercept surveys with login modal open

## Acceptance Criteria
- [x] When the login modal is open, the accessibility feedback modal should be excluded from appearing to prevent any overlap or interference.
- [x] The solution should ensure that accessibility feedback can still be provided when the login modal is not active
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by aubreyarcangel** [comment]
Reqest submitted to VEO intake portal. Ticket number FF01BE6

**Comment by ATMiddleton** [comment]
Completed 12/30/24
